<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Piano</class>
 <widget class="QMainWindow" name="Piano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="key_C7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







ะก</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_D7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_E7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_G7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_F7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_A7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_B7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_A6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_B6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_D6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_G6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_F6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_C6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







ะก</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_E6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Db">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Eb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Gb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Ab">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Bb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Db2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Eb2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Gb2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Ab2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="key_Bb2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>794</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>by drammma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>891</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(210, 210, 210);</string>
    </property>
    <property name="text">
     <string>     q             w            e               r             t              y             u              i              o             p             a              s              d             f              g</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>     1             2                             3             4             5                            6             7                             8             9              0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_C5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>51</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>







C</string>
    </property>
   </widget>
   <zorder>key_C5</zorder>
   <zorder>key_C7</zorder>
   <zorder>key_D7</zorder>
   <zorder>key_E7</zorder>
   <zorder>key_G7</zorder>
   <zorder>key_F7</zorder>
   <zorder>key_A7</zorder>
   <zorder>key_B7</zorder>
   <zorder>key_A6</zorder>
   <zorder>key_B6</zorder>
   <zorder>key_D6</zorder>
   <zorder>key_G6</zorder>
   <zorder>key_F6</zorder>
   <zorder>key_C6</zorder>
   <zorder>key_E6</zorder>
   <zorder>key_Db</zorder>
   <zorder>key_Eb</zorder>
   <zorder>key_Gb</zorder>
   <zorder>key_Ab</zorder>
   <zorder>key_Bb</zorder>
   <zorder>key_Db2</zorder>
   <zorder>key_Eb2</zorder>
   <zorder>key_Gb2</zorder>
   <zorder>key_Ab2</zorder>
   <zorder>key_Bb2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
